packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -97,25 +134,28 @@ To render an initially drilled in menu, the `menuDrilledIn`, `drilldownPath`, an
 
 ### With drilldown breadcrumbs
 
+Use breadcrumbs when a drilldown menu has more than 2 levels to offer users better navigation.
+
 ```ts file="MenuWithDrilldownBreadcrumbs.tsx" isBeta
 ```
 
-### Scrollable
+### Scrollable menus
+
+Use the `isScrollable` property to make a long `<Menu>` scrollable and visually condensed.
 
 ```ts file="MenuScrollable.tsx"
 ```
 
-### Scrollable with custom menu height
+### Scrollable menu with custom height
 
-```ts file="MenuScrollableCustomMenuHeight.tsx"
-```
-
-### With footer
+Adjust the visual size of a scrollable menu by using the `menuHeight` property within `<MenuContent>`. This example adjusts the height to 200px.
 
-```ts file="MenuWithFooter.tsx"
+```ts file="MenuScrollableCustomMenuHeight.tsx"
 ```
 
 ### With view more
 
+You may add a "view more" link at the bottom of a menu, which allows users to expand long menus. In this example, 3 additional menu items are revealed each time the "view more" option is selected. Once all items are visible, the "view more" link disappears.
```

Comment:
I feel like for the most part this description is fine, but did offer an alternative below. The view more functionality isn't really a menu feature, but more logic that can be setup with the menu. This example is really just showing one way you could go about that.

```suggestion
If you want to initially render only a certain number of menu items within a large menu, you can add a "view more" menu item with a callback passed into its `onClick` property that will render additional menu items.

In this example, 3 additional menu items are revealed each time the "view more" option is selected, with a loading icon simulating a network call to fetch more items. Once all items are visible, the "view more" link disappears.
```

Basically wanted to mention the "simulated network call" as well as the fact that the logic to render more items has to be passed in/isn't built into the menu itself.

Reply:
I agree that "expandable" doesn't seem to fit, though I do get where you were going with it. "With view more" I think should be fitting enough for the purpose of the example, though it does kind of bring into question whether it's an example or more of a demo... @mcarrano what do you think?

Reply:
I agree that "expandable" is not right. I'm fine with calling this example "With view more". Or another idea is "With progressive disclosure" as I believe the label "View more" could be changed if one wanted. As for whether this is really more like a demo, I don't know. With the new composable approach I'm less clear about what's a demo vs a component example :)

Reply:
I'd agree with either "With view more" or "With progressive disclosure".

And yeah, there are several components with things that could technically fit as an example or a demo so it's tricky. I think what makes this possibly feel like a demo to me is that it isn't showcasing a particular Menu prop in itself nor does it feel like functionality that's "out of the box", but rather it's functionality that has to be custom written to achieve the desired result.

That said I'm definitely good with keeping this as an example, even if it's a "just for now" thing.